dmsg-daemon/cmd/internal/dmsg-client.go

```diff
@@ -0,0 +1,45 @@
+package internal
+
+import (
+	"context"
+	"time"
+
+	"github.com/skycoin/dmsg"
+	"github.com/skycoin/dmsg/cipher"
+	"github.com/skycoin/dmsg/disc"
+	"github.com/skycoin/skywire/pkg/skyenv"
+)
+
+// ConnClient connects to dmsg clients
+func ConnClient() error {
```

Comment:
Naming could be clearer imo. Youre instantiating / setting up client so maybe something along those lines.